lib/pure/times.nim

```diff
@@ -1288,32 +1289,20 @@ proc timeToTimeInfo*(t: Time): TimeInfo =
 
 proc timeToTimeInterval*(t: Time): TimeInterval =
```

Comment:
I think we should rename all procs that follow this naming convention. i.e. instead of ``timeToTimeInterval`` just call it ``toTimeInterval``.